modules/distr-tracing-tempo-config-spanmetrics.adoc

```diff
@@ -0,0 +1,93 @@
+////
+This module included in the following assemblies:
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-spanmetrics_{context}"]
+= Configure span RED metrics
+
+Trace data contains rich information and most importantly the data is normalized across instrumented languages and frameworks.
+Therefore, additional metrics can be extracted from traces. These metrics are request count, duration and error count (RED).
+The metrics can be visualized in Jaeger console in the Monitor tab.
+
+The metrics are derived from spans in the OpenTelemetry collector, then scraped from the collector by the Prometheus deployed in the user-workload monitoring stack.
+Jaeger UI queries these metrics from the Prometheus endpoint and visualizes.
+
+== OpenTelemetry collector configuration
+
+The OpenTelemetry collector needs to configure `spanmetrics` connector which derives metrics from traces and exports the metrics in the Prometheus format.
+
+.OpenTelemetry collector CR for span RED
+[source,yaml]
+----
+kind: OpenTelemetryCollector
+apiVersion: opentelemetry.io/v1alpha1
+metadata:
+  name: otel
+spec:
+  mode: deployment
+  observability:
+    metrics:
+      enableMetrics: true <1>
+  config: |
+    connectors:
+      spanmetrics: <2>
+        metrics_flush_interval: 15s
+
+    receivers:
+      otlp: <3>
+        protocols:
+          grpc:
+          http:
+
+    exporters:
+      prometheus: <4>
+        endpoint: 0.0.0.0:8889
+        resource_to_telemetry_conversion:
+          enabled: true # by default resource attributes are dropped
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          exporters: [otlp, spanmetrics] <4>
+        metrics:
+          receivers: [spanmetrics] <5>
+          exporters: [prometheus]
+----
+<1> Spanmetrics connector receives traces and exports metrics.
+<2> OTLP receiver to receive spans in the OpenTelemetry protocol.
+<3> Prometheus exporter is used to export metrics in the Prometheus format.
+<4> Spanmetrics connector is configured as exporter in traces pipeline.
+<5> Spanmetrics connector is configured as receiver in metrics pipeline.
+
+// TODO Max please fix this appropriately
+The setup as well requires to configure the user-workload monitoring stack to scrape the Prometheus endpoint of the collector - https://github.com/openshift/openshift-docs/pull/60953/files#diff-bd0dc510dd646c60e63a7daf0b9d22eb7525adc8467753a6bdb37948345992ddR7
```

Comment:
@max-cx this should point to this document https://github.com/max-cx/openshift-docs/pull/10/files#diff-ff751529c1b337a1d13c6471f9d002d28fba1ca7b53680b227fe755203503f4e